showcase/app/components/page-components/app-footer/sub-sections/base.gts

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import ShwTextH2 from 'showcase/components/shw/text/h2';
+import ShwTextH3 from 'showcase/components/shw/text/h3';
+import ShwTextH4 from 'showcase/components/shw/text/h4';
+import ShwTextBody from 'showcase/components/shw/text/body';
+import ShwFlex from 'showcase/components/shw/flex';
+import ShwDivider from 'showcase/components/shw/divider';
+
+import HdsAppFooterItem from '@hashicorp/design-system-components/components/hds/app-footer/item';
+import HdsAppFooterLink from '@hashicorp/design-system-components/components/hds/app-footer/link';
+import HdsAppFooterStatusLink from '@hashicorp/design-system-components/components/hds/app-footer/status-link';
+import HdsAppFooterLegalLinks from '@hashicorp/design-system-components/components/hds/app-footer/legal-links';
+import HdsAppFooterCopyright from '@hashicorp/design-system-components/components/hds/app-footer/copyright';
+import HdsTextBody from '@hashicorp/design-system-components/components/hds/text/body';
+
+import { on } from '@ember/modifier';
+import style from 'ember-style-modifier';
+
+import { array } from '@ember/helper';
+import { concat } from '@ember/helper';
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
```

Comment:
[nit]
we tend to have this on the top of the imports, so when looking at the file one can immediately see if it's a template-only component or not